app/jobs/work_process_extractor_job.rb

```diff
@@ -0,0 +1,13 @@
+class WorkProcessExtractorJob < ApplicationJob
+  queue_as :default
+
+  def perform()
+    file = '/Users/nina/Documents/ApprenticeshipStandardsDotOrg/tmp/test4.pdf'
+    reader = PDF::Reader.new(file)
+    text = reader.pages.map{|page| page.text}
```

Comment:
Where are we getting the file from?

Reply:
These would be all of the ActiveStorage::Attachment files that are linked to the SourceFile records.